geopyspark/geotrellis/__init__.py

```diff
@@ -258,20 +258,38 @@ def _asdict(self):
 GlobalLayout.__new__.__defaults__ = (256, None, 0.1)
 
 
-LocalLayout = namedtuple("LayoutDefinition", 'tile_size')
```

Comment:
`LocalLayout` can still be constructed from `tile_size` https://github.com/jbouffard/geopyspark/blob/feature/layout-type-local-layout/geopyspark/geotrellis/__init__.py#L288 It just won't have it as a property.